packages/gestalt/src/NestedList.js

```diff
@@ -0,0 +1,57 @@
+// @flow strict
+import { type Node } from 'react';
+import classnames from 'classnames';
+import styles from './List.css';
+import getChildrenToArray from './List/getChildrenToArray.js';
+import { ListProvider, useList } from './contexts/ListProvider.js';
+import { NestingProvider } from './contexts/NestingProvider.js';
+
+type ListType = 'bare' | 'ordered' | 'unordered';
+
+type Props = {|
+  /**
+   * Use List.Item to build nested lists. See [subcomponents](/web/list#Subcomponents).
```

Comment:
Ditto here on using the full URL, and perhaps linking to the "nesting" variant instead